pom.xml

```diff
@@ -300,6 +298,19 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.coveo</groupId>
+                <artifactId>fmt-maven-plugin</artifactId>
```

Comment:
I'm still having the Issue where even if I run this plugin I still need to run the spotless plugin to get the build to pass.